### DISCUSSION TITLE
Eliminate debug output from bimmerconnected

### DISCUSSION THREAD

**Discussion by borisneubert** [bimmerconnected/bimmer_connected#594] [Feature request] [answered]
I run bimmerconnected from cron every 5 minutes to log the position of my car. bimmerconnected creates the following output:

`DEBUG:asyncio:Using selector: EpollSelector`

which is mailed by cron and fills my mailbox. I turned logging off by adding the following lines to `bin/bimmerconnected` after import sys:

```
import logging
logging.getLogger('asyncio').setLevel(logging.WARNING)
```

I suggest to include this into the standard.

**Comment by rikroe** [comment, marked as answer]
Sorry taking that long to answer, but 
a) got fixed with #598
b) all output is forced to JSON if you provide the `--json` flag
